crates/provider/src/ext/mod.rs

```diff
@@ -3,6 +3,11 @@
 mod admin;
 pub use admin::AdminApi;
 
+#[cfg(feature = "anvil")]
+mod anvil;
+#[cfg(feature = "anvil")]
```

Comment:
this feature pulls in node-bindings, which i don't love doing just for using the anvil API. Should we rename this flag to `anvil-api` and the other flag to `anvil-node`